packages/phone-number-privacy/signer/src/database/wrappers/account.ts

```diff
@@ -41,6 +41,7 @@ export async function incrementQueryCount(account: string) {
       await accounts()
         .where(ACCOUNTS_COLUMNS.address, account)
         .increment(ACCOUNTS_COLUMNS.numLookups, 1)
+      return true
```

Comment:
This fixes the super small logging error. It wasn't going to break anything, but might have led to confusion while debugging.